### PR TITLE
Flutter Ninjas Tokyo 2024

### PR DESCRIPTION
**Pull request by kboy-silvergym**
There are still a lot of TBA...! But I'm very sure this gonna be held in 2024

### CODE REVIEW

**Comment by marandaneto** [comment]
Thanks @kboy-silvergym 
Let's keep it as a draft until there's a date otherwise, it's not gonna be shown on the website anyway.

**Comment by kboy-silvergym** [comment]
@marandaneto Finally, the date has been decided!!

File: _conferences/flutter-ninjas-tokyo-2024.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+location: Tokyo, Japan
+online: false
+
+date_start: 2024-06-13
+date_end:   2024-06-14
+
+cfp:
+  start: TBA
+  end:   TBA
+  site: TBA
```
Comment (marandaneto) [inline _conferences/flutter-ninjas-tokyo-2024.md]:
```suggestion
  start: 2024-06-13
  end:   2024-06-14
  site: https://twitter.com/FlutterNinjas
```

**Comment by marandaneto** [comment]
> @marandaneto Finally, the date has been decided!!

we can fix the CfP and URL once its out, thanks.

**Comment by kboy-silvergym** [comment]
>we can fix the CfP and URL once its out, thanks.

OK!

**Pull request merged**